easybuild/easyblocks/b/boost.py

```diff
@@ -89,6 +89,8 @@ def patch_step(self):
     def configure_step(self):
         """Configure Boost build using custom tools"""
 
+        configopts = "--prefix=%s" % self.objdir
```

Comment:
Right, but how will this work in case multiple configurations are specified in the easyconfig?
```{.py}
configopts = [
  " --shared",  # first build
  " --static"    # second build
]
```
Though this is not an issue right now for the Boost configs.